contrib/labstack/echo.v4/appsec_test.go

```diff
@@ -271,7 +271,6 @@ func TestControlFlow(t *testing.T) {
 			},
 			handler: func(echo.Context) error {
 				panic("unexpected control flow")
-				return nil
```

Comment:
@rafaeljusto Although it's a minor change, I wouldn't do it in this PR. It's unrelated.

Reply:
Yes you can revert as it creates linter warnings in some editors IIRC

Reply:
Reverting this change will remove the need for appsec approval a well. Do ping us in case for some reason you still need it 👍

Reply:
Oh sorry, I will revert it. I couldn't resist a "code unreachable" warning in the same package 😄  But yeah, it's not related to the change. I will rebase the commits to avoid having this in the commit history.